src/main/java/model/input/file/DataFile.java

```diff
@@ -251,4 +304,64 @@ public void setFirstRowAsHeader(boolean firstRowAsHeader) {
 	public String toString() {
 		return "[" + path + "]";
 	}
+
+	/**
+	 * Sets the standard metadata value.
+	 *
+	 * @param name The name that the metadata column will get
+	 * @param type The type of the column
+	 */
+	public void createMetaDataValue(String name, String type) throws ClassNotFoundException {
```

Comment:
Aangezien deze public is, zou ik het mooier vinden als de type binnenkomt als een class. Waarschijnlijk is er een reden dat dit niet kan. Wat is deze?